src/ConfigEditor/Auth/auth-method/AuthMethodSettings.tsx

```diff
@@ -75,9 +77,22 @@ export const AuthMethodSettings: React.FC<Props> = ({
         return acc;
       }, {}) ?? {};
 
+    const preparedDefaultOptions = {} as Record<AuthMethod, SelectableValue<AuthMethod>>;
+    let k: keyof typeof AuthMethod;
```

Comment:
do we have to declare k here?

Reply:
I know you can't type it in the for...in loop, but that doesn't mean there isn't a better way to do it :)